relayer/chains/evm/listener.go

```diff
@@ -219,7 +170,7 @@ func (p *Provider) FindMessages(ctx context.Context, lbn *types.BlockNotificatio
 }
 
 func (p *Provider) GetConcurrency(ctx context.Context, startHeight, currentHeight uint64) int {
-	diff := int(currentHeight - startHeight)
+	diff := int(currentHeight-startHeight/p.cfg.BlockBatchSize) + 1
```

Comment:
Please add extra parenthesis to make the computation more clear and readable.

Reply:
optimization. loss for readability.

Reply:
@debendraoli dai, which one you intend to do?
1. 10-5/5 = 9 or 
2. (10-5)/5 = 1

first one does not makes sense.

Reply:
You have to convert the cfg option to int again.

Reply:
I admit that unnecessary conversions lead to higher CPU usage, but the concern here is the calculation seems wrong. If you intend to get the result like that of second option then we need parenthesis. And clearly, I don't think you are intending to get result like that of first option.

Reply:
> You have to convert the cfg option to int again.
> 
> These minor improvements I made led to significant performance gains, not by smaller margin, whopping ~100%.

It's hard to believe removing parenthesis would give you 100% performance gains. Can you elaborate a bit more .

Reply:
I was taking overall conversions and calculation into account not only with this change, big numbers conversion actually takes toll on cpu.

Reply:
> I admit that unnecessary conversions lead to higher CPU usage, but the concern here is the calculation seems wrong. If you intend to get the result like that of second option then we need parenthesis. And clearly, I don't think you are intending to get result like that of first option.

Yeah, makes sense it does wrong calculation. Thanks.

I thought you're talking about readability.

Reply:
I meant when we need to convert to uint to int if we care about readability.